packages/archway-proto/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Where this license file comes from (was it auto-generated by some tool)? We usually apply the Apache 2.0 license to our projects, and there's already one LICENSE file in the repository root.